tests/Unit/Filament/Resources/Locations/LocationResourceTest.php

```
@@ -0,0 +1,67 @@
+    public function should_return_correct_navigation_icon(): void
+    {
+        $resource = new LocationResource;
+        $iconProperty = new \ReflectionProperty(LocationResource::class, 'navigationIcon');
+
+        $this->assertSame(Heroicon::OutlinedHomeModern, $iconProperty->getDefaultValue());
+    }
+
+    #[\PHPUnit\Framework\Attributes\Test]
+    public function should_return_correct_record_title_attribute(): void
+    {
+        $resource = new LocationResource;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$resource'. (undefined)

(UnusedLocalVariable)